js/slideshow.js

```diff
@@ -109,9 +112,12 @@
 					this.errorLoadingImage = false;
 					this.currentImage = img;
 
-					var backgroundColour = '#000';
+					var backgroundColour = this.darkBackgroundColour;
 					if (transparent) {
-						backgroundColour = '#fff';
+						backgroundColour = this.lightBackgroundColour;
+					}
+					if (image.backGroundColour !== null) {
```

Comment:
The `image.backGroundColour` should always be defined. If you still want to be sure, you can do:
```js
var backgroundColour = this.darkBackgroundColour;
if (image.backGroundColour !== null) { // lowercase g would be better
    backgroundColour = image.backGroundColour;
}
```